lib/resources/r4/condition.yaml

```diff
@@ -29,7 +27,6 @@ fields:
 
 - name: clinicalStatus
   required: 'Yes'
```

Comment:
double check both instances of clinicalStatus here, and category and code below

Reply:
Through a combination of making calls and reading the validators, I have came to the conclusion that the 'required' field is correct on these. However for this clinicalStatus the action is set to a terminology, so having it be required kinda makes sense, even though I don't think that does anything.

Reply:
i think we should remove the key entirely in these cases if it is not used

Reply:
[Removed required field from terminologies](https://github.com/cerner/fhir.cerner.com/pull/374/commits/b2c52fced44727f05e6d5b7bafd68cbbb969ced7)